bundles/AdminBundle/Controller/Admin/PortalController.php

```diff
@@ -327,21 +330,21 @@ public function portletModifiedObjectsAction(Request $request)
             'limit' => 10,
             'order' => 'DESC',
             'orderKey' => 'o_modificationDate',
-            'condition' => "o_userModification = '".$this->getAdminUser()->getId()."'",
         ]);
 
+        $user = Admin::getCurrentUser();
+        $list->filterAccessibleByUser($user, 'view');
+
         $response = [];
         $response['objects'] = [];
 
         foreach ($list as $object) {
-            if ($object->isAllowed('view')) {
-                $response['objects'][] = [
-                    'id' => $object->getId(),
-                    'type' => $object->getType(),
-                    'path' => $object->getRealFullPath(),
-                    'date' => $object->getModificationDate(),
-                ];
-            }
+            $response['objects'][] = [
```

Comment:
shouldn't we keep that check for extra security? we also have them in the tree controllers...

Reply:
Let's discuss in above conversation. When we find another method there, I will apply it also here.